packages/schemas/src/gen/index.ts

```diff
@@ -44,8 +44,7 @@ const generate = async () => {
             const name = normalizeWhitespaces(prefix).split(' ')[2];
             assert(name, 'Missing table name: ' + prefix);
 
-            const fields = removeParentheses(body)
-              .split(',')
+            const fields = splitColumnDefinitions(body)
```

Comment:
It takes me a while to figure out what "column" represents for

Reply:
better `splitTableFieldDefinitions`?

Reply:
TableField or TableColumn? Well, let's leave it to @gao-sun , both are fine for me tho.